[2025-05-28T18:31:22.729+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mrp_colors_csv_to_parquet.sleep_1 manual__2025-05-28T18:31:12.735402+00:00 [queued]>
[2025-05-28T18:31:22.740+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mrp_colors_csv_to_parquet.sleep_1 manual__2025-05-28T18:31:12.735402+00:00 [queued]>
[2025-05-28T18:31:22.741+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-05-28T18:31:22.762+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): sleep_1> on 2025-05-28 18:31:12.735402+00:00
[2025-05-28T18:31:22.783+0000] {standard_task_runner.py:57} INFO - Started process 182 to run task
[2025-05-28T18:31:22.814+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mrp_colors_csv_to_parquet', 'sleep_1', 'manual__2025-05-28T18:31:12.735402+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/pl_spark_mrp_colors.py', '--cfg-path', '/tmp/tmpji2z6lpf']
[2025-05-28T18:31:22.820+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask sleep_1
[2025-05-28T18:31:22.952+0000] {task_command.py:410} INFO - Running <TaskInstance: mrp_colors_csv_to_parquet.sleep_1 manual__2025-05-28T18:31:12.735402+00:00 [running]> on host 138222addb2d
[2025-05-28T18:31:23.133+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mrp_colors_csv_to_parquet' AIRFLOW_CTX_TASK_ID='sleep_1' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T18:31:12.735402+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-28T18:31:12.735402+00:00'
[2025-05-28T18:31:23.141+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-28T18:31:23.147+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 10']
[2025-05-28T18:31:23.173+0000] {subprocess.py:86} INFO - Output:
[2025-05-28T18:31:33.135+0000] {local_task_job_runner.py:292} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-05-28T18:31:33.147+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 182. PIDs of all processes in the group: [183, 182]
[2025-05-28T18:31:33.150+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 182
[2025-05-28T18:31:33.154+0000] {taskinstance.py:1517} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-05-28T18:31:33.162+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2025-05-28T18:31:33.329+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=182, status='terminated', exitcode=0, started='18:31:22') (182) terminated with exit code 0
[2025-05-28T18:31:33.330+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=183, status='terminated', started='18:31:22') (183) terminated with exit code None
